### PR TITLE
[Fix ] otherHero component on tablet && desktop

### PR DESCRIPTION
**Pull request by nerdlet**
## Description
Fixes # 179685152

## Type of change
- [x] Bug fix (non-breaking change which fixes an issue)

## Screenshots

##Current

Screenshot 2021-09-24 at 18 28 47

Screenshot 2021-09-24 at 18 29 33

##Fix
Screenshot 2021-09-24 at 18 26 50

Screenshot 2021-09-24 at 18 27 23

## Checklist:

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation

### CODE REVIEW

File: src/components/OtherHero/index.js [hunk 1]
```diff
@@ -77,6 +85,11 @@ const useStyles = makeStyles(({ palette, typography, breakpoints }) => ({
       height: "100%",
       marginLeft: `calc((-100vw + ${breakpoints.values.md}px)/6)`,
     },
+    [breakpoints.up("lg")]: {
+      position: "relative",
+      height: "100%",
+      marginLeft: `calc((-100vw + ${breakpoints.values.md}px)/4.1)`,
```
Comment (kilemensi) [inline src/components/OtherHero/index.js:91]: Where does this `4.1` come from?

**Review by kilemensi — COMMENTED** [review]
👍🏾 

---

In tablet mode the right image still shows outside the section
Screenshot from 2021-09-27 10-48-45

**Review by KobbyMmo — COMMENTED** [review]
looks good @nerdlet 

I was also thinking if we could have the same effect by changing the dom structure instead. Like moving that image into the section component.
it shifts just a bit on screen sizes above 3000px, but aside from that, it's good.

**Comment by nerdlet** [comment]
> looks good @nerdlet
> 
> I was also thinking if we could have the same effect by changing the dom structure instead. Like moving that image into the section component.
> it shifts just a bit on screen sizes above 3000px, but aside from that, it's good.

Ahhh, okay, I did try this before but had some layout issues with the `rightBackgroundImage` shifting a bit more to the left...on md and above devices. Um, let me try one more time...I think i can reorganize the layout to handle that

**Review by kilemensi — COMMENTED** [review]
👍🏾 @nerdlet

I think we might be playing a Whac-A-Mole game with this PR. While desktop looks perfect, tablet now has issues.
Screenshot 2021-09-30 121805

What is the **root** cause? Is it the images or the component design itself that we need to rethink?

**Comment by nerdlet** [comment]
> 👍🏾 @nerdlet
> 
> I think we might be playing a Whac-A-Mole game with this PR. While desktop looks perfect, tablet now has issues. Screenshot 2021-09-30 121805
> 
> What is the **root** cause? Is it the images or the component design itself that we need to rethink?

@kilemensi um, I think it is the component design... not necessarily the images. Let me take another look and see how else ic can improve it

**Review by kilemensi — APPROVED** [review]
🚀 

-----

Still doesn't align perfectly in tablet but lets focus on high priority tasks first... We may need to re-implement this component should we get time later on.
Screenshot 2021-10-01 at 10-55-28 PesaYetu

**Pull request merged**